target-app/Home/ViewModels/HomeViewModel.swift

```diff
@@ -10,9 +10,27 @@ import CoreLocation
 
 class HomeViewModel {
     
+    // MARK: - Observed Properties
+    @Published private var state: AuthViewModelState = .network(state: .idle)
```

Comment:
is it ok to be of type AuthViewModelState?  Also if this is a private var, the published annotation doesn't make much sense since it is not used within this class either.